ui/app/routes/vault/cluster.js

```diff
@@ -82,6 +83,9 @@ export default Route.extend(ModelBoundaryRoute, ClusterRoute, {
   },
 
   model(params) {
+    // if a user's browser settings block localStorage they will be unable to use Vault. The method will throw the error and the rest of the application will not load.
+    // localStorage.isLocalStorageSupported();
```

Comment:
Are we not actually checking if it's supported here?

Reply:
Oh, I just started testing on this branch. Thinking that something was wrong with the non-manual backport. I'll close this.